janus/lib/client/jsx/etna.d.ts

```diff
@@ -0,0 +1,153 @@
+// TODO: In the future, we'd want to convert etna to typescript, or atleast type its .d.ts files there, rather
```

Comment:
Basically a slightly modified version of the one from Vulcan. Would definitely reduce copy / paste if we add in a standard version to `etna-js` itself...

Reply:
Yeah, I want to work on getting etna-js to typescript sometime soon.